resources/lang/en/resources.php

```diff
@@ -9,10 +9,15 @@
     */
 
     'page_title' => 'Voting Resources',
-    'tranlations_in_progress' => '', 
+    'tranlations_in_progress' => '',
     'general_section_title' => 'General Resources',
     'voting_section_title' => 'Voting Registration',
     'kansas_section_title' => 'Kansas Resources',
+    'elected_officials' => [
+        'name' => 'Find Your Elected Officials',
+        'link' => route('elected-officials.index'),
+        'page_intro' => 'For any U.S. residential address, you can look up who represents that address at each elected level of government. '
+    ],
```

Comment:
Let's get these copied into `es/resources.php` as well. I typically just use google translate, and then have Claudia, our Spanish language partner, review things and let me know of any corrections.